jenkins.sh

```diff
@@ -36,10 +40,18 @@ shellcheck ./**/*.sh
 
 # Find all jenkins.sh's and run them.
 find . -mindepth 2 -maxdepth 5 -name jenkins.sh -type f | while read path; do
+  dir="${path%/jenkins.sh}"
+  app_version="jenkins-${dir//[^a-z]/}"
   (
-  pushd "${path%jenkins.sh}"
-  /bin/bash ./jenkins.sh
+  pushd "$dir"
+  # Need different app versions because flex can't deploy over an existing
+  # version
+  GOOGLE_VERSION_ID="$app_version" /bin/bash ./jenkins.sh
+  echo "Return code: $?"
+
+  # Clean up the app version in the background
+  delete_app_version "$app_version" &
```

Comment:
You might want to [nohup](http://stackoverflow.com/questions/15595374/whats-the-difference-between-nohup-and-ampersand) this, actually. That way if the script terminates, it'll finish the clean-up script.